components/common/src/command/package/install.rs

```diff
@@ -57,6 +57,7 @@ pub const RETRY_WAIT: u64 = 3000;
 
 pub fn start<P1: ?Sized, P2: ?Sized>(ui: &mut UI,
                                      url: &str,
+                                     channel: Option<&str>,
```

Comment:
Would it be worth making a type alias or a newtype to represent a `Channel`? I'm thinking ahead to if/when we need to validate what chars constitute a legal name, etc. Maybe a `Cow`?